samples/ipc_call_interface/src/consumer.c

```diff
@@ -2,8 +2,8 @@
 // Licensed under the MIT license.
 // See LICENSE file in the project root for full license information.
 
-#include "math_tlb.h"
 #include "az_ulib_result.h"
+#include "math_client.h"
```

Comment:
I think naming this "consumer.h" would clarify the separation of the caller's implementation of the ipc_calls of the math interface and the publisher's implementation of the math interface itself. Also moving to seperate interface and caller repos would help (as I mentioned before).

Reply:
consumer.c and .h are customer code once the math class, for both provider and consumer are autogenerated. Let me reorganize the code in directories and see if it became cleaner. 

---
In reply to: [529833668](https://github.com/Azure/azure-ulib-c/pull/53#discussion_r529833668) [](ancestors = 529833668)